packages/loader/src/types.ts

```diff
@@ -0,0 +1,44 @@
+import type { GenesisCommit, LoadOpts } from '@ceramicnetwork/common'
+import type { ModelInstanceDocument } from '@ceramicnetwork/stream-model-instance'
+import type { CommitID, StreamID } from '@ceramicnetwork/streamid'
+
+export type DocID = CommitID | StreamID | string
```

Comment:
are there risks from letting this key be a StreamID or a string?  Feels like we would want to pick one and always coerce inputs into that?  What if you try to load the same stream, once via StreamID and again with a string, will they actually be treated as the same LoadKey and wind up in the same slot in the cache?

Reply:
Yes, this is handled by this conversion function used by the cache: https://github.com/ceramicstudio/js-composedb/pull/207/files#diff-99f92e68738a8038a38d74537507f6b2e8831d31ba89f5f637a57c0c7d1dbff8R55-R57